guides/source/configuring.md

```diff
@@ -637,7 +637,7 @@ Is a boolean value which controls whether or not timestamping of `create` and `u
 
 #### `config.active_record.partial_inserts`
 
-Is a boolean value and controls whether or not partial writes are used when creating new records (i.e. whether inserts only set attributes that are different from the default). The default value is `true`.
+Is a boolean value and controls whether or not partial writes are used when creating new records (i.e. whether inserts only set attributes that are different from the default). The default value is `false`.
```

Comment:
Update "For '7.0'" section instead.

https://github.com/rails/rails/blob/3beb0f12e16109d5fa95f5a636759e0d3a86e3f1/guides/source/configuring.md#L1691-L1706

Reply:
@kamipo I have updated this configuration(**`config.active_record.partial_inserts`: `false`**) in the **"For '7.0'" section** as well.

Reply:
```suggestion
Is a boolean value and controls whether or not partial writes are used when creating new records (i.e. whether inserts only set attributes that are different from the default). The default value is `true`.
```

Reply:
Hi @kamipo 
I am a bit confused here. Why we are not updating the default value of the `partial_inserts` here?

Because as per the [activerecord changelog](https://github.com/rails/rails/blob/main/activerecord/CHANGELOG.md) and code **`load_default`**(loads the configuration for rails) method https://github.com/rails/rails/blob/53000f3a2df5c59252d019bbb8d46728b291ec74/railties/lib/rails/application/configuration.rb#L236 the default value for the **`config.active_record.partial_inserts`** should be **`false`** instead of **`true`**. 🤔

Reply:
`config.active_record.partial_inserts` is `false` only for the `load_default "7.0"` app, existing apps are still `true`.

https://github.com/rails/rails/blob/d3f4db9e95a475822c02b29241a5d07cbcff6fd5/activerecord/lib/active_record/attribute_methods/dirty.rb#L17-L18

Reply:
Got it @kamipo Thanks!!